test/integrated/lsms.jl

```diff
@@ -61,7 +61,7 @@ using ClimaLSM
         return update_aux!
     end
 
-    function ClimaLSM.make_set_initial_aux_state(m::DummyModel1{FT}) where {FT}
+    function ClimaLSM.make_set_initial_aux_state(m::DummyModel{FT}) where {FT}
```

Comment:
typo? should this be changed?

Reply:
With the 1 there, there are two definitions of make_set_initial_aux_state for the type `DummyModel1` and none for `DummyModel`, and the tests fail... But I am not sure why the same tests pass on main

Reply:
Hm, just a definition of make_update_aux and make_set_initial_aux_state for DummyModel1. For DummyModel, we are using the default (defined in src/shared_utilities/model.jl).

Reply:
Oh yes you're right it is not a duplicate definition, I can't read.  One of the tests still fails if this is `DummyModel1` though, and I don't know why. I will reset this back to what it was before and see if I can figure out what's going on.

Reply:
Ok well I must have an issue with my local manifest or something because after changing it back it still fails locally on my machine but passes here.